src/fpm/manifest.f90

```diff
@@ -89,7 +90,7 @@ end subroutine default_test
 
 
     !> Obtain package meta data from a configuation file
-    subroutine get_package_data(package, file, error, apply_defaults)
+    subroutine get_package_data(package, file, error, apply_defaults, add_is_windows_macro)
```

Comment:
I don't like the idea of polluting the package constructor with something specific to compiling fpm, maybe it would belong in `fpm_filesystem` or similar, but I'm fine with that if it's not a problem for the other reviewers.